capitaliq/src/main/scala/com/quantemplate/capitaliq/commands/revenuereport/cmdName.scala

```diff
@@ -0,0 +1,3 @@
+package com.quantemplate.capitaliq.commands.revenuereport
+
+val revenueReportCmdName = "generateRevenueReport"
```

Comment:
isn't more elegant to create companion object for class revenueReportCmd and put it there?

Reply:
I guess so, but I encountered some circular dependencies along the way so I moved it out of the companion object. I can investigate it further later.